chapters/ch09/2.embedding-based-personalization.ipynb

```diff
@@ -1011,26 +935,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "KMeans Predicted Cluster:\n",
-      "    44 (Microwave_Cu._Ft._Stainless-Steel_Oven)\n",
+      "    10 (Microwave_Cu._Ft._Over-the-Range_Stainless-Steel)\n",
```

Comment:
Uh oh... what happened with these? The scikit learn downgrade from 1.5.2 to 1.2.1 (committed to main on Oct 15) should have made these deterministic with the previous values  (`44 (Microwave...`, etc.).

Since this branch was verified on Nov 18, that implies either that this notebook wasn't re-run or that the scikit learn downgrade didn't work to make these deterministic and consistent with what's in the manuscript. I'm assuming for now that this notebook wasn't re-run (at least that's my hope!).

Can you please verify this notebook again and, assuming the code generates the values in the manuscript, please update this file with the verified version.

Reply:
I've re-run these to be Solr, the output here is from Opensearch. 

Solr's is still consistent